_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,165 @@
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the first [non-empty](#non-empty) HTML `title` element in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element.
+
+### Expectation
+
+Each target element describes the topic or purpose of the overall content of the [document](https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+- This rule assumes that browsers only recognize the first `title` element if multiple `title` elements are present in the [document](https://www.w3.org/TR/dom/#concept-document). Testing shows that this in general is the case. Therefore the scope of this rule is limited to only checking the first `title` element in a document.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.4.2: Page Titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)
+- This rule is testing [Technique G88: Providing descriptive titles for Web pages](https://www.w3.org/WAI/WCAG21/Techniques/general/G88)
+- A prerequisite for this rule is that [Technique H25: Providing a title using the title element](https://www.w3.org/WAI/WCAG21/Techniques/html/H25) is used for living up to 2.4.2 Page Titled
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+The ´title´ element describes the content of the document.
+
+```html
+<html>
+  <head>
+    <title>Clementine harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+#### Passed example 1
+
+Two <title> tags where the first one describes the content of the document.
+
+```html
+<html>
+  <head>
+    <title>Clementine harvesting season</title>
+    <title>Apple harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+#### Passed example 3
+
+Even though the descriptive `title` element is not placed within the `head` element that is the context the element can be used in [according to the HTML specification](https://www.w3.org/TR/html/document-metadata.html#the-title-element), the rule still passes, since the browser fixes it, and it doesn't cause any known accessibility issues.
+
+```html
+<html>
+  <head>
+  </head>
+  <body>
+    <title>Clementine harvesting season</title>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+### Failed
+
+#### Failed example 1
+
+`title` is not descriptive of the content of the document.
+
+```html
+<html>
+  <head>
+    <title>Apple harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
```

Comment:
I think a failed test case should be added with a second title element.

```
### Failed example 2

Even though a correct `title` element is put in the head of the document, this rule only looks at the first (non-empty) title element.

```html
<html>
  <head>
    <title>Apple harvesting season</title>
    <title>Clementine harvesting season</title>
  </head>
  <body>
    <p>Clementines will be ready to harvest from late October through February.</p>
  </body>
</html>
```
```